MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift

```diff
@@ -158,22 +127,9 @@ extension ActionToolbarContainer {
         case reply
         case reblog
         case like
-        case bookmark
```

Comment:
What's the need to remove this?

Reply:
The button no longer exists so I removed the dead code.